index.html

```diff
@@ -0,0 +1,300 @@
+                        <i class="fa-brands fa-twitter team__soc-icon"></i>
+                    </div>
+                </div>
+            </div>
+
+            <div class="team__card">
+                <div class="team__img-container">
+                    <img src="./assets/img/t3.png" alt="" class="team__img">
+                </div>
+                <div class="team__infos">
+                    <div class="team__name">Avie Beaton</div>
+                    <div class="team__job">CO Founder</div>
+                    <div class="team__soc">
+                        <i class="fa-brands fa-square-facebook team__soc-icon"></i>
+                        <i class="fa-brands fa-instagram team__soc-icon"></i>
+                        <i class="fa-brands fa-twitter team__soc-icon"></i>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="contact container">
+        <div class="c-title c-title--xs c-title--left">Contact Us</div>
+        <div class="c-title c-title--left">Make an Appointment</div>
+        <div class="contact__form">
+            <div class="contact__inputs flex-content">
+                <input type="text" class="contact__input" placeholder="Full Name">
+                <input type="email" class="contact__input" placeholder="Email address">
+            </div>
+            <div class="contact__inputs flex-content">
+                <input type="text" class="contact__input" placeholder="Please Select">
+                <input type="text" class="contact__input" placeholder="4:00 Available">
+            </div>
+            <textarea name="" id="" cols="30" rows="10" class="contact__input contact__input--textarea" placeholder="Message"></textarea>
+        </div>
+        <button class="btn btn--default">Book Appointment</button>
+    </div>
+    <div class="contact-bar container">
+        <div class="flex-content">
+            <div class="contact-bar__texts">
+                <div class="c-title c-title--text-md c-title--left">Consulting Agency For Your Business</div>
+                <div class="c-subtitle c-subtitle--sm c-subtitle--left">the quick fox jumps over the lazy dog</div>
+            </div>
+            <button class="btn btn--default">Contact Us</button>
+        </div>
+    </div>
+
+    <footer class="footer">
+        <div class="footer__content container">
+            <div class="footer__top">
+                <div class="footer__logo">KMarketer</div>
+                <div class="footer__soc">
+                    <i class="fa-brands fa-square-facebook footer__soc-icon footer__soc-icon--fac"></i>
+                    <i class="fa-brands fa-instagram footer__soc-icon footer__soc-icon--insta"></i>
+                    <i class="fa-brands fa-twitter footer__soc-icon footer__soc-icon--twi"></i>
+                    <i class="fa-brands fa-youtube footer__soc-icon footer__soc-icon--you"></i>
```

Comment:
you can change `soc` to something else like `socials` to make it more descriptive and less confusing.